ruby_fundamentals/ruby_basics.md

```diff
@@ -0,0 +1,127 @@
+## Ruby Basics
+
+Let's try to run some Ruby code!
+
+
+- Open a terminal
+
+```
+irb
+```
+
+##### Basic example on how to print text
+```
```

Comment:
Make sure to add `ruby` para marecognize na Ruby code. Same for the others in this file 😸

Reply:
Ayyy. Ou nga haha
Okiii, will change :)